docker-compose.yaml

```diff
@@ -10,3 +10,21 @@ services:
       - ./mssql:/mssql
     ports:
       - "15785:1433"
+    healthcheck:
+        # The OPENCoronaTempTables database is initialised last, so check that
+        # we can connect to that db specifically
+        test: /opt/mssql-tools18/bin/sqlcmd -C -S localhost -U sa -P "Your_password123!" -d "OPENCoronaTempTables" -Q "SELECT 1" || exit 1
+        interval: 10s
+        timeout: 3s
+        retries: 10
+        start_period: 10s
+
+  prod-test:
+    build:
+      context: .
+    entrypoint: scripts/run-tests.sh
```

Comment:
I slightly feel this would be more intentional as
```
entrypoint: ""
command: "scripts/run-tests.sh"
```
but that would be longer...